frontend/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -267,23 +273,20 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
 
         const uploadInfo = {
           uploadId,
-          organization: datasetId.owningOrganization,
-          name: datasetId.name,
-          layersToLink: [],
```

Comment:
When `layersToLink` is omitted, it simply defaults to `[]`, right?

Reply:
I removed the field here because the backend now saves the information from the `reserveUpload` request. The frontend always sends an empty list there https://github.com/scalableminds/webknossos/pull/5958/files/57f7c21ff54f1a25cd443404e2a796fa50681095#diff-2060cc6fbf0debf8ec6f63bfdd187937f7b1ded7d5c031522616362cd9f17149L250